resumebuilder/src/App.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './App.css';
+import Resume from './components/Resume'
+import applicant from './resume/applicant'
+
+function App() {
+  return (
+    <div className="App">
+      {/* <header className="App-header"> */}
+        <>
```

Comment:
Since you have a parent component (the div), you don't need to use a Fragment as well!

Reply:
I think the same and also (just an opinionated way of thinking) if you are going to use a Fragment prefer the explicit <Fragment /> since it's more readable!

Reply:
Thanks guys! Im embarassed! I didnt check App.js once more before finishing up where I was at! Appreciate the fragment feedback, I love them haha

Reply:
Hey don't worry so much about it @7FigureSwagger we only had 2 hours you did an amazing work in that time frame!